nestkernel/connection_manager.cpp

```
@@ -413,7 +413,8 @@ nest::ConnectionManager::connect( index sgid,
     // we do not allow to connect a device to a global receiver at the moment
     if ( not source->has_proxies() )
     {
-      return;
+      throw IllegalConnection( "The models " + target->get_name() + " and "
```

should we throw a similar error in line546?